inngest/src/main/kotlin/com/inngest/Inngest.kt

```diff
@@ -21,37 +19,28 @@ class Inngest
 
         internal val httpClient = HttpClient(RequestConfig(headers))
 
-        inline fun <reified T> send(payload: Any): T? =
-            sendEvent<T>(payload) lambda@{ response ->
+        /**
+         * Send a single event to Inngest,
+         *
+         * @param event The event to send.
+         *
+         */
+        fun send(event: InngestEvent): SendEventsResponse? = send(arrayOf(event))
+
+        /**
+         * Send multiple events to Inngest,
+         *
+         * @param events The events to send.
+         *
+         */
+        fun send(events: Array<InngestEvent>): SendEventsResponse? {
```

Comment:
Hm, I can't find a concrete reference for this but I think Kotlin `Array` is the equivalent of a java array, which means `InngestEvent[]` which I think tends to be less flexible than the `List` interface, since that has multiple implementations and `[]` arrays can easily be made into `ArrayList`

Some SO posts that seem to align with that I'm saying, but nothing definitive
- https://stackoverflow.com/a/70628712
- https://stackoverflow.com/a/716619

Reply:
let's address this in a follow up